tests/Fixtures/FakeGraphApiForResumableUpload.php

```diff
@@ -57,25 +59,22 @@ public function send($url, $method, $body, array $headers, $timeOut)
     private function respondStart()
     {
         if ($this->respondWith == 'FAIL_ON_START') {
-            return new GraphRawResponse(
-                "HTTP/1.1 500 OK\r\nFoo: Bar",
+            return new Response(500, ['Foo'=>'Bar'],
```

Comment:
Please fix CS issue, this should be:
```php
return new Response(
    500,
    ['Foo' => 'Bar'],
    '...'
);
```
You can check [this build](https://travis-ci.org/facebook/php-graph-sdk/jobs/167553016) for all CS issues